### PR TITLE
Deploy/sblumin/1270045305 packagejson

### PR DESCRIPTION
**Pull request by sblumin**
### Administrative Info

Monday Board ID: 851212620
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes

What changes did you make?

Update uri from localhost to heroku to prepare for Heroku deployment

### Testing

How did you confirm your changes worked?

Will test out webapp after deployment🤞

- TODO

### Confirmation of Change

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| [TODO Screenshot Title] |
| ----------------------- |
| [TODO Screenshot Link]  |

### CODE REVIEW

File: frontend/src/pages/Login/Login.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { setJWT, setUser } from "../../auth";
 import "./Login.css";
 
 const config = require("../../config");
+require("dotenv").config();
```
Comment (wllmwu) [inline frontend/src/pages/Login/Login.js:8]: I think one or both of these lines isn't needed anymore

**Review by wllmwu — COMMENTED** [review]
Looks LGTM to me! I think there's just a couple remaining lines that we can remove since we're not using the config thing anymore

**Review by ecflores30 — APPROVED** [review]
Looks great to me!

**Review by wllmwu — APPROVED** [review]
Looks LGTM to me! 👌

**Pull request merged**